README.md

```diff
@@ -573,14 +573,20 @@ k3os:
 ### `k3os.k3s_args`
 
 Arguments to be passed to the k3s process.  The arguments should start with `server` or `agent`
-to be valid.
+to be valid. `k3s_args` is an exec-style (aka uninterpreted) argument array, this means that k3os is invoking k3s in two seperate arguments like so:
 
-Example
+```bash
+#!/bin/sh
+exec "k3s" "server" "--cluster-cidr 10.107.0.0/23" "--service-cidr 10.107.1.0/23"
```

Comment:
`exec "k3s" "server" "--cluster-cidr 10.107.0.0/23" "--service-cidr 10.107.1.0/23"`

This is exactly the kind of thing folks have been tripping over. It should be either of the below:
- `exec "k3s" "server" "--cluster-cidr=10.107.0.0/23" "--service-cidr=10.107.1.0/23"`
- `exec "k3s" "server" "--cluster-cidr" "10.107.0.0/23" "--service-cidr" "10.107.1.0/23"`

Reply:
This is incorrect here, but correct below.

Reply:
Ah ok missed that part! I'll make that edit, thanks man :)